### PR TITLE
Preset tooltip

### PR DESCRIPTION
**Pull request by kd0aij**
add a tooltip explaining what the preset entry field does

### CODE REVIEW

File: src/me/drton/flightplot/FlightPlot.java [hunk 1]
```diff
@@ -729,6 +729,11 @@ public void showImportPresetDialog() {
                 Preset preset = Preset.unpackJSONObject(new JSONObject(new String(b, Charset.forName("utf8"))));
                 loadPreset(preset);
                 processFile();
+                if (preset.getTitle().equals("")) {
+                    // generate a namestring
+                    preset.setTitle(file.getName().substring(0, file.getName().indexOf(".fplot")));
+                }
+                presetComboBox.addItem(preset);
```
Comment (AndreasAntener) [inline src/me/drton/flightplot/FlightPlot.java]: If added to the list, one might make the assumption that this entry is somehow connected with the file you imported. I don't think it's good to mix these ATM. Maybe it needs a bit of rethinking in the direction of merging these 2 ways of handling presets.

Comment (AndreasAntener) [inline src/me/drton/flightplot/FlightPlot.java]: Although I can see the practicality '). At least, the imported preset should become the selected item in the list if added.

**Comment by kd0aij** [comment]
until I made these changes, I never saw any entries in the preset pulldown and had no clue that typing something in that field would do anything. Perhaps because the export preset menu function leaves the Title as the null string? How do you use the preset pulldown list, and where are those presets stored? They don't appear in the same location as those saved using the export preset function.

**Comment by AndreasAntener** [comment]
These presets are stored in the application preferences along with all the other stuff (e.g. window positions, export settings).

Usage: You can just enter a name and hit [enter], it will then store the preset under the entered name. If you exit FlightPlot, it will save these presets in the application preferences.

Export/import is more for sharing presets, and I never use it ;)

**Comment by kd0aij** [comment]
I've always used the import/export functions, since I didn't know how the pulldown worked. And to get them to appear in the preset list, I must import each one, then type a name for it in the list...

Is the format of saved presets identical to the exported format?

And I hope you at least test export/import periodically :)

**Comment by AndreasAntener** [comment]
The serialization format is the same, but the application preferences are hidden in the platform dependent user home somewhere, and mixed with all the other settings, so not really usable for actual export/import.

**Comment by kd0aij** [comment]
The pulldown is much more convenient than import/export, but something caused my list of 20 or so to disappear recently, and it's empty again :(
And that's after I deleted all my exported presets...

**Comment by AndreasAntener** [comment]
oh, this happened to me too at some point, but only a while ago. Not sure what caused it

**Comment by kd0aij** [comment]
good incentive to tie the 2 together though; automatically export when saving a preset from the pulldown. Not sure what the advantage is of hiding them away.

**Comment by DrTon** [comment]
Agree, hiding is not useful here. Maybe we just need to save them to some config folder and use simple "file == preset" storage. I also had some problems with Java preferences. Disappearing may be caused by changing storage format some time ago, sorry for this :)

Presets interface was influenced by "saved selections" tool in 3ds max. I'm open to suggestions how to make it more intuitive but still minimalistic and easy to use.

**Comment by DrTon** [comment]
Cherry picked tooltip text, thanks. Closing this. For new ideas about presets please open new issue.

**Pull request closed without merging**